lib/jsdom/living/nodes/HTMLInputElement-impl.js

```diff
@@ -232,6 +232,11 @@ class HTMLInputElementImpl extends HTMLElementImpl {
       if (form) {
         form._doSubmit();
       }
+    } else if (this.type === "reset") {
+      const { form } = this;
+      if (form) {
```

Comment:
Can you deduplicate the form-related lines with the ones in the submit condition code above, making it similar to the code in HTMLButtonElement?

Reply:
This should be fixed. It is formatted a little different than HTMLButtonElement because of the additional "checkbox"/"radio" condition, but at least no longer initializing the same constant in 2 different places.